fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/fluid/FluidStorageUtil.java

```diff
@@ -111,7 +111,7 @@ private static boolean moveWithSound(Storage<FluidVariant> from, Storage<FluidVa
 						if (!fill && handItem == Items.POTION) sound = SoundEvents.ITEM_BOTTLE_EMPTY;
 					}
 
-					player.playSound(sound, SoundCategory.BLOCKS, 1, 1);
+					player.getWorld().playSound(player, player.getX(), player.getEyeY(), player.getZ(), sound, SoundCategory.PLAYERS, 1, 1);
```

Comment:
Has this been tested, I believe you need to pass null as the first param otherwise this is going to play for everyone BUT the player.

Reply:
This method should be called both on client and server. `ClientWorld.playSound` will play sound to the `except`, while `ServerWorld.playSound` will play sound to all players but the `except`. This usage is according to vanilla usage, such as `FenceGateBlock#onUse`.

Reply:
Humm, I tested with the chute block in the FAPI test mods, they only call this on the server. I wonder if its worth wrappning this is a `!world.isClient`? Maybe @Technici4n would have some input on this?

Reply:
Well it depends on which side modders are expected to call it. It's probably worth documenting.

Reply:
Yes, you were right, it makes sense to call this on both sides. I have gone ahead and updated the testmod, this now works as expected. 👍

Reply:
Should we indicate it needs to be called on both sides in the doc?